word_search.py

```diff
@@ -0,0 +1,102 @@
+'''
+Created on 9 June 2017
+
+@author: Eva
+'''
+from queue import Queue
+
+class WordSearch:
```

Comment:
The use of a `class` here seems not helpful.
What is a `WordSearch`? It is not a concept like the `BinaryTree` of the previous assignment.
 
The method `words_found_in_grid` uses `self` to store what are essentially local variables.
The method `addWordToSearch` could receive the things it uses for `self` as arguments.
Being able to store the values on `self` might seem convenient at first, but actually makes it
a bit harder to understand and has disadvantages.

Do you think we can try to make the methods top-level functions outside the class?